pymongo/errors.py

```diff
@@ -18,14 +18,15 @@
 
 try:
```

Comment:
How about moving this to ssl_support instead?

Reply:
It can't go there because CertificateError because then there would be a circular import ssl_support -> pyopenssl_context -> ssl_support